core/Files.h

```diff
@@ -107,14 +107,17 @@ class File final {
     const std::string source_;
     mutable std::shared_ptr<std::vector<int>> lineBreaks_;
     mutable StrictLevel minErrorLevel_ = StrictLevel::Max;
-    std::shared_ptr<const FileHash> hash_;
 
 public:
     const StrictLevel originalSigil;
     StrictLevel strictLevel;
 
     const CompiledLevel compiledLevel;
+
+private:
+    std::shared_ptr<const FileHash> hash_;
```

Comment:
This is not relevant to the size changes, right? It's just a "happen to notice"?

Reply:
This is moving the location of the `hash_` member within the class so that `originalSigil`, `StrictLevel`, and `compiledLevel` pack into the same word as `minErrorLevel_`.  `hash_` was `private` before and is `private` after (unless I screwed something up).